radio/src/storage/yaml/yaml_datastructs_funcs.cpp

```diff
@@ -1637,6 +1638,7 @@ static bool w_customFn(void* user, uint8_t* data, uint32_t bitoffs,
       if (!wf(opaque, str, strlen(str))) return false;
     }
   }
+
   if (!wf(opaque, "\"", 1)) return false;
   return true;
```

Comment:
No part of this PR, but this type of formatting should be omitted. The risk of confusing the next person is too high.